README.md

```diff
@@ -17,4 +17,5 @@ Possible environments:
 * __BACKEND_URL__ - url address of backend server. It's can't be alias for your ip (for example shoov.local) because docker doesn't know about your hosts. 
 * __DEBUG__ - enable debug mode, output all docker logs to stdout. _(optional)_
 * __VNC_PASSOWRD__ - password for connect to VNC. _(optional)_
-* __TIMEOUT_LIMIT__ - timeout in seconds to start silenium container _(optional)_, default 30 seconds.
+* __TIMEOUT_LIMIT__ - timeout in __seconds__ to start silenium container _(optional)_, by default 30 seconds.
+* __UPTIME_LIMIT__ - timeout in __minutes__ for maximum server uptime _(optional)_, by default 20 minutes.
```

Comment:
`DOCKER_RUN_TIMEOUT: Determines the time in seconds which the docker may execute the tests. Defaults to 1200 seconds (20 minutes).`